src/main/java/org/osc/manager/ism/entities/SGInterfaceElement.java

```diff
@@ -0,0 +1,88 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.osc.sdk.manager.element.ManagerPolicyElement;
+import org.osc.sdk.manager.element.SecurityGroupInterfaceElement;
+
+public class SGInterfaceElement implements SecurityGroupInterfaceElement {
+
+    private Long id;
+
+    private String name;
+
+    private Set<PolicyEntity> policies = new HashSet<>();
+
+    private String tag;
+
+    private SecurityGroupEntity securityGroup;
+
+    public SGInterfaceElement() {
```

Comment:
do you need this public constructor?

Reply:
Not needed, I borrowed the code from else where and did not notice it. will remove it.